content/infrastructure/livecontainers.md

```diff
@@ -9,13 +9,13 @@ customnav: infrastructurenav
 
 ## Introduction
 
-Datadog Live Containers allows for real-time visibility of all of the containers across your environment.  Taking inspiration from bedrock tools like `htop` and `ctop`, this centralized view, combined with existing tagging capabilities, allows you to understand what is going on at any level of your system and drill all the way down into the most fine details.
+[Datadog Live Containers](https://app.datadoghq.com/containers) allows for real-time visibility of all of the containers across your environment.  Taking inspiration from bedrock tools like `htop` and `ctop`, this centralized view, combined with existing tagging capabilities, allows you to understand what is going on at any level of your system and drill all the way down into the most fine details.
 
 {{< img src="infrastructure/livecontainers/LiveContainersWithSummaries.png" >}} 
 
 ## Installation
 
-Live Containers has been introduced in Datadog Agent version 5.17.2.  After updating to a recent version of the Agent, no other configuration is necessary.
+[Live Containers](https://app.datadoghq.com/containers) has been introduced in Datadog Agent version 5.17.2.  After updating to a recent version of the Agent, no other configuration is necessary.
```

Comment:
After updating to this version of the Agent or later, no other configuration is necessary.